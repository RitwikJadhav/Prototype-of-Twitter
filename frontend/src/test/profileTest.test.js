import React from 'react';
import Enzyme, { mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ProfileLikes from '.././components/profile-module/profileLikes';
import renderer from 'react-test-renderer';

Enzyme.configure({ adpater: new Adapter() });

it('should render Profile Page correctly', () => {
    const component = renderer.create(<ProfileLikes />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});