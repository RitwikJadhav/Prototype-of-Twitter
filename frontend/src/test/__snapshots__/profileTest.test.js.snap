// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Profile Page correctly 1`] = `
<div
  className="row profile-section"
>
  <div
    className="row"
  >
    <h2
      className="error-msg col-sm-12"
    >
      You have not liked any tweets.
    </h2>
    <h2
      className="error-msg-2 col-sm-12"
    >
      When you do, they will show up here.
    </h2>
  </div>
</div>
`;
